src/gui/properties/speedplotview.h

```diff
@@ -86,6 +90,36 @@ class SpeedPlotView : public QGraphicsView
     void paintEvent(QPaintEvent *event) override;
 
 private:
+    class Averager
+    {
+    public:
+        Averager(int _divider, boost::circular_buffer<PointData> &_sink)
+            : cnt(0)
+            , divider(_divider)
+            , sink(_sink)
+        {acc.reset();};
```

Comment:
multiline
And I think the last semicolon is superfluous

Reply:
fixed